tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -10,7 +10,7 @@
 
   <mksrf_flai>
     <entry>
-      <data_filename>lnd/clm2/rawdata/CTSM54RawData/globalctsm53histMKSRFDeg025_250417/mksrf_landuse_ctsm53_pftlai_CLIM.c250417.nc</data_filename>
+      <data_filename>lnd/clm2/rawdata/CTSM54RawData/CLM6_LUH3_HIST_CMIP7/mksrf_pftlai_clm6_histLUH3_2005_c251012.nc</data_filename>
```

Comment:
@ekluzek the rest of these are the changes needed to pick up Peter's new raw datasets.